src/main/java/com/teamrocket/tms/controllers/ProjectController.java

```diff
@@ -0,0 +1,47 @@
+package com.teamrocket.tms.controllers;
+
+import com.teamrocket.tms.models.dtos.ProjectDTO;
+import com.teamrocket.tms.services.project.ProjectService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/projects")
+public class ProjectController {
+
+    private final ProjectService projectService;
+
+    public ProjectController(ProjectService projectService) {
+        this.projectService = projectService;
+    }
+
+    @PostMapping
+    public ResponseEntity<ProjectDTO> createProject(@RequestBody ProjectDTO projectDTO) {
```

Comment:
There are validation annotations in ProjectDTO but no @Valid for the request body